### PR TITLE
QA. SHUF-87 create usecase on TitledAccentInfo

### PR DESCRIPTION
**Pull request by naveenpardeep**

### CODE REVIEW

**Comment by Igorpi25** [comment]
Rename branch according to https://azart.atlassian.net/wiki/spaces/SD/pages/7962940/GIT

**Comment by Igorpi25** [comment]
This element works as part of a list. Can we generate just one golden PNG containing multiple elements within a list (no-content, long-content)? It could reduce the number of generated PNG files. What do you thing?

**Comment by Igorpi25** [comment]
I don't see the **critical use-case** when the width is extremely short, like 300px. We must write tests not only where the element is known to work well, but we should cover bt tests cases where the element has a chance to fail.

File: example/test/lib/ui_kit/molecules/chips/titled_AccentInfo_golden_test.dart
Comment (Igorpi25) [inline example/test/lib/ui_kit/molecules/chips/titled_AccentInfo_golden_test.dart:1]: Why do you have only one use case in your code, while there are several generated PNG files?

Comment (naveenpardeep) [reply example/test/lib/ui_kit/molecules/chips/titled_AccentInfo_golden_test.dart:1]: because i changed same file code ...next time i will create new file

**Pull request closed without merging**